extensions/README.md

```diff
@@ -15,8 +15,10 @@
 |     15     | [64 Bit Operations + Registers](./quad-word-operations)   | None          | Mostly Stable     |
 |     16     | [32 Bit Address Space](./32-bit-address-space)            | CP1.14        | Under Development |
 |     17     | Virtual Memory + 32 Bit Paging                            | CP1.16, CP2.2 | Planned           |
-|     32     | 64 Bit Address Space                                      | CP1.15        | Planned           |
-|     33     | Virtual Memory + 64 Bit Paging                            | CP1.32, CP2.2 | Planned           |
+|     32     | [64 Bit Address Space](./64-bit-address-space)            | CP1.15        | Under Development |
+|     33     | Virtual Memory + 64 Bit Paging (48 bit VA)                | CP1.32, CP2.2 | Planned           |
+|     34     | Virtual Memory + 64 Bit Paging (57 bit VA)                | CP1.32, CP2.2 | Planned           |
+|     35     | Virtual Memory + 64 Bit Paging (64 bit VA)                | CP1.32, CP2.2 | Planned           |
```

Comment:
Are we sure these are the exact plans we have? If not, I wouldn't list them here.

Reply:
You're right, I forgot about the 16 bit paging extension. I'm pretty sure the plan is to stay have similar paging modes as x86 and x86-64. The main difference is that we're skipping 32 bit PAE and we're defining 64 bit paging which goes all the way to the 64 bit address space limit. The reason for these different levels is runtime performance. Each increase in size requires an additional page table layer.

16 bit: 1 layer
32 bit: 2 layers
64 bit (48): 4 layers
64 bit (56): 5 layers
64 bit (64): 6 layers

Reply:
from discord: https://discord.com/channels/828292123936948244/946806826756882552/1108708220563894332
[this page](https://developer.arm.com/documentation/den0024/a/The-Memory-Management-Unit/Translation-tables-in-ARMv8-A/AArch64-descriptor-format) and the next one describe page tables in aarch64. They can be 3 or 4 levels. So I think it would make more sense if 16/32/64 bit paging were incompatible (better pedagogically as it keeps the one you want to focus on simpler). Then the 64 bit ones could be 2/3/4 layers. And we probably only need to support one granule size.

At any rate, that's a planning thing and doesn't affect this PR.

Reply:
I'm not sure how they'd be compatible in the first place aside from using similar page table entry structures.